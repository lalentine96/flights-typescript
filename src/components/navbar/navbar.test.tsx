import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

import Navbar from './navbar';

afterEach(cleanup);

it('should match the snapshot', () => {
    const { asFragment } = render(
        <BrowserRouter>
            <Navbar 
                onSearchChange={() => {}}
                onToggleCheckbox={() => {}}
            />
        </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
});