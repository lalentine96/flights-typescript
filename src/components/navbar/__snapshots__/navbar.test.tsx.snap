// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="navbar navbar-expand-sm bg-info navbar-dark"
  >
    <ul
      class="navbar-nav mr-auto"
    >
      <li
        class="nav-item"
      >
        <a
          class="nav-link"
          href="/arrivals"
        >
          Arrivals
        </a>
      </li>
      <li
        class="nav-item"
      >
        <a
          class="nav-link"
          href="/departures"
        >
          Departures
        </a>
      </li>
    </ul>
    <form
      class="form-inline"
    >
      <label
        class="form-check-label text-white mr-3"
      >
        <input
          class="form-check-input"
          type="checkbox"
        />
        Show delayed only
      </label>
      <input
        class="form-control nav-input"
        placeholder="Search by flight number"
        type="text"
      />
    </form>
  </div>
</DocumentFragment>
`;
